modules/plugin/imagemosaic/src/main/java/org/geotools/gce/imagemosaic/granulecollector/BaseSubmosaicProducer.java

```diff
@@ -83,6 +83,8 @@
 
     protected Color inputTransparentColor;
 
+    // boolean forceFootprints = JAIExt.;
```

Comment:
Is it here for "debugging/testing" purposes?

Reply:
Yep, must be removed